### DISCUSSION TITLE
[Config Support]: Cannot seem to get started

### DISCUSSION THREAD

**Discussion by b24home** [blakeblackshear/frigate#10177] [Config Support] [answered]
### Describe the problem you are having

Hello. I cannot seem to get Frigate running.

I am running HA supervised and I installed via the add-ons, then via HACS. 

When I go to launch Frigate, it says that the plug-in is not running, and I click to start it. Then it loads for a few seconds, and I get the same message. It's a loop.

So I uninstalled. Then tried again. Now, when I go to HACS (after installing via the Add-On) I use the provided URL but it says Failed to Connect. 

I am NOT an advanced user, and I find the documentation very lacking (for me). It does mention in various places (different documentation) the following:

"You must create a config file as frigate.yml in the root of your Home Assistant configuration directory."
...and
"In addition, MQTT must be enabled in your Frigate configuration file and Frigate must be connected to the same MQTT server as Home Assistant for many of the entities created by the integration to function."

I don't have a clue what this means. I have spent an hour searching for help to no avail. I have MQTT set up, but I need a step by step walkthrough if this step is what is holding me back. Please help!

TYIA!!!

P.S. I simply intend to use Frigate (I think) for getting my Ring cameras and my new Tapo cameras into HA working properly. I tried Scrypted, but gave up when I coudn't figure that out and my trial expired. Currently, MQTT is pulling in my Ring cameras, but I am having trouble. Will I disable MQTT for all of my Ring devices when I have Frigate, or just the cameras, or nothing at all? What about Scrypted? Do I need that?

### Version

0.13.2

### Frigate config file

```yaml
Don't know
```

### Relevant log output

```shell
Don't know
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

CPU (no coral)

**Comment by supermarkert** [comment, marked as answer]
It seems like you didn't create the config file. As it states, you need to create this file manually in the root of your Config folder. Everything you need to know is here:  
https://docs.frigate.video/configuration/

The next page gives an example of all the configuration options (you should start with a minimal config, and go from there):
https://docs.frigate.video/configuration/reference

**Reply by 18923** [reply]
Just want to add...  Frigate will not start if the "frigate.yaml" file is invalid. If it does not start for you check the "log" file for the addon to help in resolving the problem.

**Comment by b24home** [comment]
Yup, that did it! Thank you! That specific document should definitely be linked to when the config is mentioned!!

Can you just point me in the right direction for my next steps? 

I mentioned...

> P.S. I simply intend to use Frigate (I think) for getting my Ring cameras and my new Tapo cameras into HA working properly. I tried Scrypted, but gave up when I coudn't figure that out and my trial expired. Currently, MQTT is pulling in my Ring cameras, but I am having trouble. Will I disable MQTT for all of my Ring devices when I have Frigate, or just the cameras, or nothing at all? What about Scrypted? Do I need that?

All I'm looking to do in the short term is get working cameras... then stored video... then eventually, motion triggers... and maybe one day, object detection... all of this is very complicated for a beginner, though.
